serve-web/src/Service/Availability/DatabaseAvailability.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace App\Service\Availability;
+
+use Doctrine\ORM\EntityManager;
+
+class DatabaseAvailability extends ServiceAvailabilityAbstract
+{
+    private $em;
+
+    public function __construct(EntityManager $em)
+    {
+        $this->em = $em;
+    }
+
+
+    public function ping()
```

Comment:
awesome stuff. I don't know serve very well and not sure if it's cos all the API stuff hasn't been moved to golang, but if the golang api stuff is acting as the backend then normally you'd have this in the api app but tbh it's prolly that not everything has been moved across?